### PR TITLE
08 capsule display info

### PR DESCRIPTION
**Pull request by MaximePikss**

### CODE REVIEW

**Review by RoxaneDrelon — DISMISSED** [review]
SnE

File: src/components/capsulefull/CapsuleFull.jsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+const CapsuleFull = () => {
+  const classes = Styles();
+  const { capsules } = useContext(ApiContext);
+  const { id } = useParams();
+
+  const uniqueCapsuleById = _.uniqBy([...capsules], 'capsule_id');
+
+  return (
+    <>
```
Comment (LoicBrassart) [inline src/components/capsulefull/CapsuleFull.jsx]: Useless Fragment, isn't it ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One minor Evil

**Review by RoxaneDrelon — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**